financial-connections/src/main/java/com/stripe/android/financialconnections/features/common/SharedPartnerAuth.kt

```diff
@@ -289,7 +290,7 @@ private fun LoadedContent(
 private fun PrePaneContent(
     showInModal: Boolean,
     content: OauthPrepane,
-    loading: Boolean,
+    authenticationStatus: Async<Action>,
```

Comment:
Nit: We should be consistent with `status` vs `action`.

Reply:
good point -> now the status wraps an action: https://github.com/stripe/stripe-android/pull/7943/commits/08e47e95afdffd4a7cc62117c8262239e09317b4